### DISCUSSION TITLE
Does anyone have a referral code for Heurist API Early Access?

### DISCUSSION THREAD

**Discussion by tommaay** [elizaOS/eliza#498] [Q&A]
Hi I filled out the form for Heurist api access and want to ask if anyone has a referral code so that I can get prioritized. I would love to get access and start experimenting with it and Eliza. Thanks!

**Comment by tsubasakong** [comment]
plz use `ai16z` as the referral code :)

**Comment by yodamaster726** [comment]
Thank you whoever implemented the heurist implementation - it just worked.  Thanks for the info here specially - it fastpathed the approval.
